features/aanschrijfwijze.feature

```diff
@@ -410,56 +436,53 @@ Functionaliteit: Als gemeente wil ik de juiste en consistente aanschrijfwijze va
       | type                | RaadpleegMetBurgerservicenummer |
       | burgerservicenummer | 999992934                       |
       | fields              | naam.aanschrijfwijze            |
```

Comment:
naam.aanschrijfwijze.naam ?
Dit komt op meerdere plekken voor., dus of de aansc=preekvorm toevoegen of de fields aanscherpen.
(Overigens zat dit er al in  en is dit dus niet a.g,v deze PR